### PR TITLE
Update Molde Generator Usage

### PR DESCRIPTION
**Pull request by robertomiranda**
Including has_secure_token generator

### CODE REVIEW

File: railties/lib/rails/generators/rails/model/USAGE [hunk 1]
```diff
@@ -82,6 +82,11 @@ Available field types:
     has_secure_password, you should specify `password:digest`:
 
         `rails generate model user password:digest`
+        
+    If you require a `token` string column for use with
+    has_secure_token, you should specify `token:token`:
+
+        `rails generate model user token:token`
```
Comment (vipulnsward) [inline railties/lib/rails/generators/rails/model/USAGE]: `token:token` might be a bit confusing, how about using `auth_token:token` here?

**Comment by vipulnsward** [comment]
@robertomiranda doc PR's should have `[ci skip]` in commit message so that we don't run tests on travis. 
See more- http://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html

**Comment by senny** [comment]
@robertomiranda @vipulnsward thanks guys :yellow_heart: . Made a couple minor modifications in the merge commit.

**Comment by robertomiranda** [comment]
@vipulnsward I'll keep on mind, thanks :blush:

**Pull request merged**